speechmatics/cli.py

```diff
@@ -576,12 +576,12 @@ def parse_args(args=None):
     )
     transcribe_subparser.add_argument(
         "--max-delay-mode",
-        default="flexible",
         choices=["fixed", "flexible"],
         type=str,
         help=(
             "How to interpret the max-delay size if speech is in the middle "
             "of an unbreakable entity like a number."
+            "Default choice is flexible."
```

Comment:
I think this is slightly misleading, as the flexible endpointing is not supported by older versions. And if the default gets changed on the server side in the future, this will no longer be true. I'd simply omit this information, as in this case the default is something out of control of the client and is documented in the API documentation.

Reply:
makes sense yeah.